### PR TITLE
Changing the max_queue_size directive

### PR DESCRIPTION
**Pull request by dtheodoro**
A little patch to use the configuration directive max_queue_size from file instead the default 16K

### CODE REVIEW

File: lib/sensu/extensions/handlers/relay.rb [hunk 1]
```diff
@@ -121,17 +121,21 @@ class Endpoint
     # space of a single loop tick.
     MAX_QUEUE_SIZE = 16384
 
-    attr_accessor :connection, :queue
+    attr_accessor :connection, :queue, :max_queue_size
 
-    def initialize(name, host, port)
+    def initialize(name, host, port, queue_size = nil)
```
Comment (grepory) [inline lib/sensu/extensions/handlers/relay.rb:126]: You can save the if nil check below by defaulting to MAX_QUEUE_SIZE instead of nil.

**Comment by grepory** [comment]
If you wouldn't mind making that minor change and squashing, that would be awesome. Thanks.

File: lib/sensu/extensions/handlers/relay.rb (hunk 1 above)
Comment (dtheodoro) [inline lib/sensu/extensions/handlers/relay.rb:126]: I made this way because it didn't pass the travis test suite .

Comment (grepory) [inline lib/sensu/extensions/handlers/relay.rb:126]: Strange.

**Comment by grepory** [comment]
Closing in favor of https://github.com/grepory/wizardvan/pull/2 -- thank you, though.

**Pull request closed without merging**